java-sdk/src/main/java/com/adobe/target/edge/client/ondevice/DefaultRuleLoader.java

```diff
@@ -63,12 +63,15 @@ public synchronized void start(final ClientConfig clientConfig) {
         if (started) {
             return;
         }
-        ObjectMapper mapper = new JacksonObjectMapper();
+
         byte[] artifactPayload = clientConfig.getOnDeviceArtifactPayload();
+
         if (artifactPayload != null) {
-            String payload = new String(artifactPayload, StandardCharsets.UTF_8);
-            OnDeviceDecisioningRuleSet ruleSet = mapper.readValue(payload, new GenericType<OnDeviceDecisioningRuleSet>() {});
+            ArtifactObfuscator artifactObfuscator = new ArtifactObfuscator();
```

Comment:
If I make `artifactObfuscator` a member of the class, I observed that the value is always null at line 70 in debugger.  In fact all the class members have null when running the test.  I don't know the reason, is it because of Mockito?  Any idea?